webpack.config.ts

```diff
@@ -15,8 +15,14 @@ import FixStyleOnlyEntriesPlugin from 'webpack-remove-empty-scripts';
 import IntegrationDocsFetchPlugin from './build-utils/integration-docs-fetch-plugin';
 import LastBuiltPlugin from './build-utils/last-built-plugin';
 import SentryInstrumentation from './build-utils/sentry-instrumentation';
+import {extractIOSDeviceNames} from './scripts/extract-ios-device-names';
 import babelConfig from './babel.config';
 
+// Runs as part of prebuild step to generate a list of identifier -> name mappings for  iOS
+(async () => {
+  await extractIOSDeviceNames();
```

Comment:
how terrible would it be to just commit the result to our repo or make our own package. Running this here seems a little weird

Reply:
oh wait you did commit it. maybe just leave it as a script in package.json?

Reply:
i guess its probably pretty fast...

Reply:
Yeah, this is pretty fast right now. Tbh, I just didnt want to maintain our own package... The reason the file is commited to the repo is that folks may run the build locally and it would pop up as a changed file (also why I run the output through prettier). Ideally, we could "freeze" or repo wide ignore a file once it's checked into the repo and that would be perfect for this use-case, but I didnt find a good way to do it!